src/Parser.ts

```diff
@@ -94,20 +128,31 @@ export class Parser {
   }
 
   consume(ttype: TokenType) {
-    this.advance()
+    while (!this.isAtEnd()) {
```

Comment:
checking in on the while statement here too

Reply:
Reposting what I wrote on the previous comment:
"I think I wrote this back when I had a less of an understanding of the difference between if and whiles. I can see now that there is no reason for it to do a looping check to see if it's at the end within this code."